lib/iris/coords.py

```diff
@@ -1649,8 +1648,8 @@ def bounds(self):
         """
         if self._bounds is not None:
             bounds = self._bounds
-            if isinstance(bounds, biggus.Array):
-                bounds = bounds.ndarray()
+            if is_lazy_data(bounds):
+                bounds = bounds.compute()
```

Comment:
Is there a better way to do this now? I stumbled across this `biggus` usage and felt obliged to address it ... apologies for being gratuitous in this PR.